Documentation/operations/system_requirements.rst

```diff
@@ -194,6 +194,27 @@ The following kernel configuration options are required for proper operation:
         CONFIG_GENEVE=y
         CONFIG_FIB_RULES=y
 
+
+⚠️ Note on CONFIG_FIB_RULES Activation
+
+On some embedded or custom Linux systems—especially when cross-compiling for 
+ARM—enabling ``CONFIG_FIB_RULES=y`` directly in the kernel ``.config`` may not be sufficient, 
+as it depends on other routing-related kernel options to be enabled.
+
+If ``CONFIG_FIB_RULES`` is missing from the final compiled kernel (e.g., not visible via cat /boot/config-*), 
+make sure the following options are also enabled:
+
+::
+
+	CONFIG_IP_ADVANCED_ROUTER=y
+	CONFIG_IP_FIB_TRIE_STATS=y
+	CONFIG_IP_MULTIPLE_TABLES=y
+	CONFIG_IP_ROUTE_CLASSID=y
+
+These options ensure that ``CONFIG_FIB_RULES`` becomes available and correctly set during kernel configuration.
+This has been validated on ARM embedded systems with a custom Linux OS, where Cilium failed to operate properly 
+until these dependencies were included.
```

Comment:
I'd skip the last sentence, it probably doesn't bring much information to the reader.